### ISSUE TITLE
Personal Dictionary Shortcuts not working

### ISSUE THREAD

**Issue by MarkJelic** [GrapheneOS/grapheneos.org#1087]
Extract:  Entering the shortcut for a personal entry in the Personal Dictionary(s) does not replace it with the full worded entry

Details: Coming from the iPhone, which has a function called Text Replacement, I was dismayed I could not find this until someone told me it was in the Personal Dictionary(s) in GrapheneOS.

But when I type in "bh" in any text field in any app, it does not get replaced with "Baulkham Hills" like it does on the iPhone.

Likewise, I had the shortcut "xx" put in the X (twitter) logo, and "bb" put in the Bitcoin symbol. None of that works on Graphene, unless I'm doing something wrong?
![Screenshot_20241222-223025](https://github.com/user-attachments/assets/a35f6e5a-18b9-4c1b-8cdf-5c63416fd9d3)

**Comment by thestinger** [comment]
You can use a different keyboard instead of AOSP keyboard such as FlorisBoard.

**Comment by MarkJelic** [comment]
> You can use a different keyboard instead of AOSP keyboard such as FlorisBoard.

Yes I've loaded up Gboard which seems to be working OK, and actually has much better autocorrection. The amount of typing errors I get with the standard AOSP keyboard is shocking.

Just hope my keystrokes aren't being logged by Google. And I'd have the same concern with any other third party keyboard, for that matter. Would prefer if the inbuilt one worked better.
